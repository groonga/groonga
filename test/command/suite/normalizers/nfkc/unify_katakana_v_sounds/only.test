#@add-substitution /NormalizerNFKC(\d*)/ "NormalizerNFKC#{ENV['NFKC'] || '100'}" "NormalizerNFKC"
normalize \
  'NormalizerNFKC("unify_katakana_v_sounds", true, \
                  "report_source_offset", true)' \
  "ヴァヴィヴヴェヴォヴ" \
  WITH_CHECKS|WITH_TYPES
#@remove-substitution /NormalizerNFKC(\d*)/
