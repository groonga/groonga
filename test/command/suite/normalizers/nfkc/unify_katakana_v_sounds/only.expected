normalize   'NormalizerNFKC("unify_katakana_v_sounds", true,                   "report_source_offset", true)'   "ヴァヴィヴヴェヴォヴ"   WITH_CHECKS|WITH_TYPES
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "バビブベボブ",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "null"
    ],
    "checks": [
      6,
      0,
      0,
      6,
      0,
      0,
      3,
      0,
      0,
      6,
      0,
      0,
      6,
      0,
      0,
      3,
      0,
      0
    ],
    "offsets": [
      0,
      6,
      12,
      15,
      21,
      27,
      30
    ]
  }
]
